docs/building-examples.rst

```diff
@@ -0,0 +1,246 @@
+Building example designs
+========================
+
+Before building any example, set the installation directory to match what you
+set it to earlier, for example:
+
+.. code-block:: bash
+   :name: export-install-dir
+
+   export INSTALL_DIR=~/opt/symbiflow
+
+Select your fpga family:
+
+.. tabs::
+
+   .. group-tab:: Artix-7
+
+      .. code-block:: bash
+         :name: fpga-fam-xc7
+
+         FPGA_FAM="xc7"
+
+   .. group-tab:: EOS S3
+
+      .. code-block:: bash
+         :name: fpga-fam-eos-s3
+
+         FPGA_FAM="eos-s3"
+
+Next, prepare the enviroment:
+
+.. code-block:: bash
+   :name: conda-prep-env
+
+   export PATH="$INSTALL_DIR/$FPGA_FAM/install/bin:$PATH";
+   source "$INSTALL_DIR/$FPGA_FAM/conda/etc/profile.d/conda.sh"
+
+Finally, enter your working Conda enviroment:
+
+.. code-block:: bash
+   :name: conda-act-env
+
+   conda activate $FPGA_FAM
+
+.. note::
+
+   If you don't know how to upload any of the following examples onto your
+   development board, please refer to the Running examples section.
+
+
+Xilinx 7-Series
+---------------
+
+Enter the directory that contains examples for Xilinx 7-Series FPGAs:
+
+.. code-block:: bash
+   :name: enter-dir-xc7
+
+   cd examples/xc7
+
+Counter test
```

Comment:
These should probably be separate files (IE one file per example).

Reply:
Do you mean that you want a README in every example directory that we somew
how pull in? This is a radically different setup and while I have nothing against it in principle I'd like to understand the reasoning so as not to generate confusion;)

Reply:
(again, I am all for prototyping this kind of change in a separate PR after we move to Sphinx)

Reply:
Most people are going to encounter the documentation via the GitHub repository first. If they land in https://github.com/SymbiFlow/symbiflow-examples/tree/master/xc7/counter_test directory, there should be instructions right there (which also link to the more complete sphinx documentation). 

There is an WIP example in the v2x repo at https://github.com/SymbiFlow/python-symbiflow-v2x/tree/master/tests/clock_mux